dev-docs/apis.adoc

```diff
@@ -51,27 +51,28 @@ Separating the API "definition" and "implementation" in this way allows us to on
 
 Writing a new v2 API may appear daunting, but additions in reality are actually pretty simple:
 
-1. *Create POJO ("Plain Old Java Object") classes as needed to represent the API's request and response*:
+1. *Agree on Endpoint Syntax*: Before implementing, developers should generate consensus around what the new API will look like.  General guidelines and conventions for the v2 API syntax are described <<v2-api-conventions.adoc,here>>.
```

Comment:
In markdown you can do:

```
1. First item
1. Second item
1. Third item
1. Fourth item 
```

Does that work in adoc?  would that make life easier?

Reply:
It definitely works in adoc.  The question I'd ask though is whether Github is smart enough to do it as well when it's previewing these asciidoc files.

That's the main way we expect these to be consumed, right?  Through the Github UI?  (i.e. we don't actually build these asciidoc files into any thing analogous to the ref-guide?)

I'll push the change and see how Github handles it; if GH doesn't do-the-right-thing I'll revert.

Reply:
Looks like GH does do the right thing 👍